scripts/apply_withdrawals.js

```diff
@@ -1,5 +1,5 @@
 const SnappAuction = artifacts.require("SnappAuction")
-const getArgumentsHelper = require("./script_utilities.js")
+const { getArgumentsHelper } = require("./script_utilities.js")
```

Comment:
Aren't these only necessary if there is more than one item being imported?

Reply:
It's necessary as soon as `modules.export` contains a dictionary instead of a single method. Since we added another method to the `./script_utilities.js` this is now the case.

Reply:
Interesting. Thanks!